Misc/NEWS.d/next/Library/2019-09-25-05-16-19.bpo-38265.X6-gsT.rst

```diff
@@ -0,0 +1 @@
+Update :func:`os.pread` to accept the length type as Py_ssize_t
```

Comment:
The main suggestion of importance here is specifying that the *length* parameter accepts `Py_ssize_t` __instead__ of `int`. 
```suggestion
Update the *length* parameter of :func:`os.pread` to accept
:c:type:`Py_ssize_t` instead of :c:type:`int`.
```

(included the correct word wrapping in the suggestion above)